gdcdictionary/schemas/subject.yaml

```diff
@@ -105,110 +80,23 @@ properties:
       PATRIC unique identifier for genome sequenced.
     type: string
 
-  ageunit:
-    description: >
-      Age units (yr/mo/days):
-    enum:
-      - Years
-      - Months
-      - Days
-    termDef:
-      - term: ageunit
-        source: NCI Thesaurus
-        term_id: C50400
-        term_version: 20.03e (Release date:2020-03-31)
-
-  city:
-    description: >
-      City
-    type: string
-    termDef:
-      - term: city
-        source: NCI Thesaurus
-        term_id: C25160
-        term_version: 20.03e (Release date:2020-03-31)
-
-  province:
-    description: >
-      the region/province of the patient
-    type: string
-    termDef:
-      - term: province
-        source: NCI Thesaurus
-        term_id: C25632
-        term_version: 20.03e (Release date:2020-03-31)
-
-  country:
-    description: >
-      Country
-    type: string
-    termDef:
-      - term: country
-        source: NCI Thesaurus
-        term_id: C25464
-        term_version: 20.03e (Release date:2020-03-31)
-
   wuhan_not_wuhan:
     description: >
       Wuhan not Wuhan
     enum:
       - "0"
       - "1"
 
-  latitude:
-    description: >
-      Latitude
-    type:
-      - string
-    termDef:
-      - term: latitude
-        source: NCI Thesaurus
-        term_id: C68642
-        term_version: 20.03e (Release date:2020-03-31)
-
-  longitude:
-    description: >
-      Longitude
-    type:
-      - string
-    termDef:
-      - term: longitude
-        source: NCI Thesaurus
-        term_id: C68643
-        term_version: 20.03e (Release date:2020-03-31)
-
   geo_resolution:
     description: >
       Geo resolution
     type: string
 
-  date_onset_symptoms:
-    description: >
-      Date onset symptoms
-    type: string
-
-  date_admission_hospital:
-    description: >
-      Date admission hospital
-    type: string
-
   date_confirmation:
     description: >
       Date confirmation
     type: string
 
-  symptoms:
-    description: >
-      Symptoms
-    type: array
-    items:
-      type: string
-    termDef:
-      - term: symptoms
-        source: NCI Thesaurus
-        term_id: C4876
-        term_version: 20.03e (Release date:2020-03-31)
-
   lives_in_wuhan:
```

Comment:
This prop should be deleted. It is being harmonized with the city property.

Reply:
done